tests/test_shtuff.py

```diff
@@ -72,3 +72,29 @@ def test_shtuff_with_bad_target_gracefully_dies(self):
         out = subprocess.run("shtuff into badreceiver 'echo foo'", shell=True, capture_output=True, encoding='utf-8')
         assert out.returncode == 1
         assert 'not found' in out.stderr
+
+    def test_shtuff_has(self):
+        receiver = pexpect.spawn("shtuff as cheezeburgerz")
+        receiver.expect('\$')
+        receiver.expect('\$')
+
+        out = subprocess.run("shtuff has cheezeburgerz", shell=True, capture_output=True, encoding='utf-8')
+        assert out.returncode == 1
```

Comment:
Do we really want to exit failure when the shell is found? It feels to me that for scripting purposes, it would be nice if the command exited 0 to indicate success here.

Reply:
For sure!  I'm not a python dev.  We changed stderr to stdout here, but didn't realize returning 1 meant error.

Reply:
Thanks! For the record, exiting 0 vs 1 isn't a Python thing, it's a Unix thing. See https://bash.cyberciti.biz/guide/The_exit_status_of_a_command =)